libraries/protocol/include/steem/protocol/smt_operations.hpp

```diff
@@ -8,23 +8,25 @@
 namespace steem { namespace protocol {
 
 /**
- * This operation elevates given steem account to SMT control account on conditions that:
- * a) the account is not SMT control account already and
- * b) successful fee transfer occurs.
+ * This operation introduces new SMT into blockchain as identified by
+ * Numerical Asset Identifier (NAI). Also the SMT precision (decimal points)
+ * is explicitly provided.
  */
-struct smt_elevate_account_operation : public base_operation
+struct smt_create_operation : public base_operation
```

Comment:
Please include the precision as well for clarity and redundancy.

Reply:
Ain't decimal places the equivalent of precision? If you want `decimal_places` field to be removed and `precision` field to be added, what type should the `precision` be, other than `uint8_t` of `decimal_places`?